src/Core/Abstractions/IAuthService.cs

```diff
@@ -31,7 +31,18 @@ public interface IAuthService
 
         Task<List<PasswordlessLoginResponse>> GetPasswordlessLoginRequestsAsync();
         Task<List<PasswordlessLoginResponse>> GetActivePasswordlessLoginRequestsAsync();
+        /// <summary>
+        /// Get a passwordless login request by id, the user needs to be authenticated
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
         Task<PasswordlessLoginResponse> GetPasswordlessLoginRequestByIdAsync(string id);
+        /// <summary>
+        /// Get a passwordless login request by id, doesn't required authentication
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="accessCode"></param>
+        /// <returns></returns>
         Task<PasswordlessLoginResponse> GetPasswordlessLoginResponseAsync(string id, string accessCode);
```

Comment:
Naming here it's tricky, because we have Request/Response from the http calls and the Request as in a created passwordless request, and it gets confusing.

Reply:
Renamed method that had response to request and added just the explanation for the unauthed.